release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -2807,6 +2807,56 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, the `Started` column was hidden because of reduced screen size. This caused the `VirtualizedTable` component to malfunction because of a missing sort function. As a result, the table sorting functionality was affected in the `PipelineRun` list pages. With this release, the table component handles missing sort functions correctly for reduced screen sizes. (link:https://issues.redhat.com/browse/OCPBUGS-57110[OCPBUGS-57110]) 
+
+* Previously, users did not configure a default masthead logo when customizing the theme. This omission caused an inconsistency in the user interface. With this release, the masthead logo displays a default option for both light and dark themes, improving the interface consistency. (link:https://issues.redhat.com/browse/OCPBUGS-57054[OCPBUGS-57054])
+
+* Previously, the cluster installation failed because of an invalid security group configuration for  a Network Load Balancer (NLB). This failure prevented the traffic from both primary subnets for bootstrapping. With this release, the security group is updated to allow traffic from both primary subnets for bootstrapping, and the cluster installation does not fail because of security group restrictions on additional primary subnets. (link:https://issues.redhat.com/browse/OCPBUGS-57039[OCPBUGS-57039])
+
+* Previously, users without project access saw an incomplete roles list because of improper API group access. As a result, users saw partial role information on the *Roles* page. With this release, users without project access cannot see an incomplete roles list on the *Roles* page. (link:https://issues.redhat.com/browse/OCPBUGS-56987[OCPBUGS-56987])
+
+* Previously, the `node-image create` command modified directory permissions and caused user directories to lose original permissions during the operation. With this release, the `node-image create` command preserves file permissions during `rsync`, and ensures that user directories maintain original permissions during the operation. (link:https://issues.redhat.com/browse/OCPBUGS-56905[OCPBUGS-56905])
+
+* Previously, the `delete` keyword in image names was allowed for `ImageSetConfiguration`, which is not supported. As a consequence, users encountered errors while mirroring images. With this release, the error for image names ending with `delete` in `ImageSetConfiguration` has been removed. As a result, users can now successfully mirror images with names ending in `delete`. (link:https://issues.redhat.com/browse/OCPBUGS-56798[OCPBUGS-56798])
```

Comment:
For clarity purposes, add a noun after each instance of `ImageSetConfiguration`.